### PR TITLE
OH2-385 | Code improvement: equality check, unused imports, missing dependencies,...

### PR DESCRIPTION
**Pull request by SilverD3**
See OH2-385.

This PR fixes 242 eslint warnings
eslint-warnings

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```
Comment (SteveGT96) [inline package.json:6]: What's the purpose of this new dependency ?

Comment (SilverD3) [reply package.json:6]: To fix this warning
npm-warning

**Comment by SteveGT96** [comment]
OK ! Please fix conflicts and then we could move on.

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**